content/sql/concepts/operators/terms/update/update.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: UPDATE
+Description: 'Changes/Updates data row entries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
```

Comment:
``` suggestion
  - 'paths/design-databases-with-PostgreSQL'
```
Use the course/path that is already available in the [catalog-content. md](documentation/catalog-content.md) and this path - 'paths/design-databases-with-postgresql' is not in the list of catalog-content.md so add this path to the catalog-content list.